python/ray/scripts/scripts.py

```diff
@@ -268,7 +268,7 @@ def dashboard(cluster_config_file, cluster_name, port):
 @click.option(
     "--internal-config",
     default=None,
-    type=str,
+    type=json.loads,
```

Comment:
What's this change for?

Reply:
Maybe it's a new bug in master. If I don't fix it, the Java tests can't start a local cluster.

Reply:
@kfstorm this seems related to your previous RP? Java CI is failing on master.